<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Friend</class>
 <widget class="QWidget" name="Friend">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>390</y>
     <width>89</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Return</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTextBrowser" name="recordBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>451</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="saveRecord">
   <property name="geometry">
    <rect>
     <x>402</x>
     <y>270</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="messageText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>451</width>
     <height>85</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="settingButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>270</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="stateLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
